app/assets/javascripts/code_listing/code_listing.ts

```diff
@@ -311,81 +351,172 @@ export class CodeListing {
               ${I18n.t("js.user_annotation.cancel")}
             </button>
             <button class="btn btn-text btn-primary annotation-control-button annotation-submission-button" type="button">
-              ${edit ? I18n.t("js.user_annotation.update") : I18n.t("js.user_annotation.send")}
+              ${annotation != null ? I18n.t("js.user_annotation.update") : I18n.t("js.user_annotation.send")}
```

Comment:
```suggestion
              ${annotation !== null ? I18n.t("js.user_annotation.update") : I18n.t("js.user_annotation.send")}
```
Search for ` == ` and ` != ` and replace them.